doc/compiled.json

```diff
@@ -21686,6 +21690,416 @@
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          },
+          "429": {
+            "$ref": "#/components/responses/429"
+          }
+        },
+        "x-code-samples": [
+          {
+            "lang": "Curl",
+            "source": "curl \"https://api.phrase.com/v2/projects/:project_id/jobs/:job_id/comments/:id?branch=my-feature-branch\" \\\n  -u USERNAME_OR_ACCESS_TOKEN"
+          },
+          {
+            "lang": "CLI v2",
+            "source": "phrase comments show \\\n--project_id <project_id> \\\n--job_id <job_id> \\\n--id <id> \\\n--branch my-feature-branch \\\n--access_token <token>"
+          }
+        ]
+      },
+      "patch": {
+        "summary": "Update a job comment",
+        "description": "Update an existing job comment.",
+        "operationId": "job_comment/update",
+        "tags": [
+          "Job Comments"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/X-PhraseApp-OTP"
+          },
+          {
+            "$ref": "#/components/parameters/project_id"
+          },
+          {
+            "$ref": "#/components/parameters/key_id"
+          },
+          {
+            "$ref": "#/components/parameters/id"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/paths/~1projects~1%7Bproject_id%7D~1jobs~1%7Bjob_id%7D~1comments/post/responses/201/content/application~1json/schema"
+                }
+              }
+            },
+            "headers": {
+              "X-Rate-Limit-Limit": {
+                "$ref": "#/components/headers/X-Rate-Limit-Limit"
+              },
+              "X-Rate-Limit-Remaining": {
+                "$ref": "#/components/headers/X-Rate-Limit-Remaining"
+              },
+              "X-Rate-Limit-Reset": {
+                "$ref": "#/components/headers/X-Rate-Limit-Reset"
+              }
+            }
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          },
+          "429": {
+            "$ref": "#/components/responses/429"
+          }
+        },
+        "x-code-samples": [
+          {
+            "lang": "Curl",
+            "source": "curl \"https://api.phrase.com/v2/projects/:project_id/jobs/:job_id/comments/:id\" \\\n  -u USERNAME_OR_ACCESS_TOKEN \\\n  -X PATCH \\\n  -d '{\"message\":\"Some message...\"}' \\\n  -H 'Content-Type: application/json'"
+          },
+          {
+            "lang": "CLI v2",
+            "source": "phrase comments update \\\n--project_id <project_id> \\\n--job_id <job_id> \\\n--id <id> \\\n--data '{\"message\": \"Some message...\"}' \\\n--access_token <token>"
```

Comment:
Not sure whether this is wrapped in `data` or you could just use `--message "comment"` directly

Reply:
Oh I think I understand
This is how is looks for `paths/comments/update.yaml` 
```- lang: CLI v2
  source: |-
    phrase comments update \
    --project_id <project_id> \
    --key_id <key_id> \
    --id <id> \
    --data '{"branch":"my-feature-branch", "message": "Some message..."}' \
    --access_token <token>
requestBody:
  required: true
  content:
    application/json:
      schema:
        type: object
        title: comment/update/parameters
        properties:
          branch:
            description: specify the branch to use
            type: string
            example: my-feature-branch
          message:
            description: Comment message
            type: string
            example: Some message...
```
And this is how it looks for `paths/job_comments/update.yaml
```
- lang: CLI v2
  source: |-
    phrase comments update \
    --project_id <project_id> \
    --job_id <job_id> \
    --id <id> \
    --data '{"message": "Some message..."}' \
    --access_token <token>
requestBody:
  required: true
  content:
    application/json:
      schema:
        type: object
        title: job_comment/update/parameters
        properties:
          message:
            description: Comment message
            type: string
            example: Some message...
```
so it should be the same

Reply:
Just checked and looks all good :+1: 

![Uploading Screenshot from 2022-06-09 15-15-34.png…]()